src/stream.jl

```diff
@@ -13,7 +13,8 @@ end
 function Stream()
     p = Ref{Ptr{Void}}()
     rt.cudaStreamCreate(p)
-    hnd = CuStream(p[])
+    ctx = CuCurrentContext()
```

Comment:
Browsing the API, I don't see how you do that. Link? (And what do we do if it's not? The closest thing I've found is `cuDevicePrimaryCtxRetain`, but that seems to have reference-counting implications.)